ui/components/forms/flavors/select/index.scss

```diff
@@ -21,4 +21,39 @@
       padding: $spacing-x-small;
     }
   }
-}
+  
+  &__container {
```

Comment:
Thanks for the changes! I hate to do this, but can you change this to a single underscore? `_`.